### ISSUE TITLE
Two blocks are being produced on the empty block interval instead of 1

### ISSUE THREAD

**Issue by danwt** [dymensionxyz/dymint#728]
Needs investigation

Report form Art:

```
Apr 27 18:29:20 ip-10-200-2-4 rollapp_evm[94749]: time="2024-04-27T18:29:20Z" level=info msg="commit synced[commit 436F6D6D697449447B5B3233382032303320373220313030203131332031333120313020313834203136203234342031363920313437203138302031343020313820313438203231332031383220353420373620323439203638203233332032303020333520353220323520343420342032333420323433203139305D3A313444327D]"
Apr 27 18:29:20 ip-10-200-2-4 rollapp_evm[94749]: time="2024-04-27T18:29:20Z" level=info msg="block created[height 5330 num_tx 0]" module=BlockManager
Apr 27 18:29:37 ip-10-200-2-4 rollapp_evm[94749]: time="2024-04-27T18:29:37Z" level=info msg="commit synced[commit 436F6D6D697449447B5B32352032323720353020362032313920323336203834203437203139203232392031343120313138203335203131342039302031343320313239203234342032313120323337203630203138352032392032313220323139203231392031393120313230203131372032343020323431203137365D3A313444337D]"
Apr 27 18:29:37 ip-10-200-2-4 rollapp_evm[94749]: time="2024-04-27T18:29:37Z" level=info msg="block created[height 5331 num_tx 0]" module=BlockManager
[9:15 PM] 
```

'at the end of each block time (i.e 1 hr) - 2 blocks are created and not 1 (even if the last block in the batch is empty)'

**Comment by danwt** [comment]
I think it's as simple as 

time t: The empty block timer elapses (1h) and a empty block gets created
time t+1m40s: the batch gets submitted https://github.com/dymensionxyz/dymint/blob/48ff85f45c65767d4cdfbc848c3e1eb66b73d04b/block/submit.go#L57-L61 and an empty block gets created in case the block from (1) had ibc

**Comment by danwt** [comment]
TODO: decide if we change it or keep as is

**Comment by mtsitrin** [comment]
will it be closed by #794 ?

**Comment by danwt** [comment]
yes I think it should fix it, since we will now only create another block ('priority') if there was a tx 👍
